config/passport.js

```diff
@@ -0,0 +1,66 @@
+const LocalStrategy = require('passport-local').Strategy
+const User = require('../models/user')
+
+module.exports = (passport) => {
+  // =========================================================================
+  // LOCAL SIGNUP ============================================================
+  // =========================================================================
```

Comment:
Let's avoid putting lines in comments. They're look "loud" and don't really add anything of value. Just `// Local signup` looks better (heck, IMO even that's not necessary, since we're already passing `'local-signup'` as the first argument to `passport.use`

Reply:
agree